x/ccv/provider/migrations/v6/migrations.go

```diff
@@ -0,0 +1,16 @@
+package v4
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+
+	providertypes "github.com/cosmos/interchain-security/v4/x/ccv/provider/types"
+)
+
+// MigrateParams adds missing provider chain params to the param store.
+func MigrateParams(ctx sdk.Context, paramsSubspace paramtypes.Subspace) {
```

Comment:
[This PR](https://github.com/cosmos/interchain-security/pull/1952/files#diff-634767bb375db45fd33c40d36b03f596cf760a84b91b78cf5a51661b0a835af3) already bumped consensus to version 6.

You can stay on consensus version 6 and change the migratinon code so it runs both migrator functions.

Reply:
Yeah, I haven't rebased.